### ISSUE TITLE
parietal cortex atrophy 

### ISSUE THREAD

**Issue by sarahstofel** [mgijax/mammalian-phenotype-ontology#3430]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

parietal cortex atrophy 

>2. Synonyms (e.g., absent spleen)

atrophy of the parietal lobe

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

acquired atrophy of the upper central part of the cerebral hemisphere

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID: 18413473

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

forebrain atrophy, abnormal parietal lobe morphology

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

https://orcid.org/0000-0003-2307-1226

**Comment by anna-anagnostop** [comment]
For atrophy terms, the MP typically uses a textual definition constructed as follows:

"acquired diminution of the size of X associated with wasting as from death and reabsorption of cells, diminished cellular proliferation, decreased cellular volume, pressure, ischemia, malnutrition, reduced function or malfunction, or hormonal changes"

where X is the anatomical entity

**Comment by sbello** [comment]
MP:0021011
reopened to discuss the definition question Anna raised, I entered this as Sarah had it since both atrophy and parietal cortex are defined elsewhere but maybe we should switch to the more consistent format.

**Comment by anna-anagnostop** [comment]
Definition switched to the more consistent format.
